rerun_py/rerun_sdk/rerun/log/camera.py

```diff
@@ -66,8 +87,50 @@ def log_pinhole(
         Specifies the [`rerun.RecordingStream`][] to use.
         If left unspecified, defaults to the current active data recording, if there is one.
         See also: [`rerun.init`][], [`rerun.set_global_data_recording`][].
+    camera_xyz:
+        Sets the view coordinates for the camera. The default is "RDF", i.e. X=Right, Y=Down, Z=Forward.
+        Other common formats are "RUB" (X=Right, Y=Up, Z=Back) and "FLU" (X=Forward, Y=Left, Z=Up).
+        Equivalent to calling [`rerun.log_view_coordinates(entity, xyz=…)`][rerun.log_view_coordinates].
+        NOTE: setting this to something else than "RDF" (the default) will change the orientation of the camera frustum,
+        and make the pinhole matrix not match up with the coordinate system of the pinhole entity.
+        The pinhole matrix (the `child_from_parent` argument) always project along the Z axis of the camera space,
+        but will be re-oritented to project along another axis if the `camera_xyz` argument is set.
```

Comment:
This definition of "re-oriented" is vague as it doesn't do a great job of describing what happens to the direction of pixel-x and pixel-y directions. I believe in essence this is more of "remapping" the axes than re-orienting (which will be important once we allow right-to-left conversions).

I believe the actual definition of this is:
 - Whichever axis is labeled F will be the positive projection direction (B means negative projection values are "into" the image).
 - Whichever axis is labeled R is positive pixel-x (increasing column index).  L means we flip the image horizontally.
 - Whichever axis is labeled D is the positive pixel-y (increasing row index). U means we flip the image vertically.

Reply:
I've expanded the docstring